scripts/schedule.coffee

```diff
@@ -0,0 +1,116 @@
+# Description:
+#   Displays Deliver current presentors, next presentors or full schedule
+#
+#
+# Dependencies:
+#  "cheerio": "^0.20.0"
+#  "needle": "^1.0.0"
+#  "async": "^1.5.2",
+#  "moment": "^2.15.1"
+#
+# Commands:
+#   hubot who is speaking now? - Displays all presenters currently speaking
+#	hubot who is speaking next? - Displays all presenters speaking next
+#	hubot schedule (day)- Displays full schedule for requested day
```

Comment:
Can you change this to (wednesday|thursday) since that's the only options it is going to be?